charts/hush-sensor/templates/sentryserviceaccount.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
   name: {{ include "hush-sensor.sentryFullName" . }}
   labels: {{- include "hush-sensor.labels" . | nindent 4 }}
   namespace: {{ include "hush-sensor.namespace" . }}
-  {{ with and .Values.sentry.serviceAccount .Values.sentry.serviceAccount.annotations -}}
-  annotations: {{ toYaml . | nindent 4 }}
+  {{- if and .Values.sentry.crawler .Values.sentry.crawler.aws (eq .Values.sentry.crawler.aws.mode "irsa") }}
+  annotations:
+    eks.amazonaws.com/role-arn: {{ .Values.sentry.crawler.aws.iam_role }}
```

Comment:
This should be merged with `.Values.sentry.serviceAccount.annotations`.
As it is written now `.Values.sentry.serviceAccount.annotations` is completely ignored.